modules/openxr/openxr_api_extension.cpp

```diff
@@ -138,6 +140,11 @@ bool OpenXRAPIExtension::can_render() {
 	return OpenXRAPI::get_singleton()->can_render();
 }
 
+uint64_t OpenXRAPIExtension::get_hand_tracker_handle(int p_hand_index) {
```

Comment:
I'm not totally sure about the "handle" in `get_hand_tracker_handle()` - we don't use that word for any of the other pointers. Why not just `get_hand_tracker()`?

Reply:
In that case the doc should also just call it "pointer".

Reply:
The `OpenXRHandTrackingExtension` has a `get_hand_tracker()` function that returns a different `HandTracker` struct. Naming the functions the same way when they return different things felt a bit off. If you feel otherwise, happy to rename it.

Reply:
Well, that class is unexposed, so no one will know about it except Godot engine developers :-)

In this classes, there's 4 other exposed methods that return pointers to structs (`get_instance()`, `get_place_space()`, etc) and none of them use the word "handle", so to me it feels inconsistent with those.

Reply:
Point taken, thanks! I've updated the names.